FABulous/FABulous_CLI/helper.py

```diff
@@ -226,6 +228,7 @@ def create_project(project_dir: Path, lang: Literal["verilog", "vhdl"] = "verilo
 
     with open(os.path.join(project_dir, ".FABulous/.env"), "w") as env_file:
         env_file.write(f"FAB_PROJ_LANG={lang}\n")
+        env_file.write(f"VERSION={version('FABulous-FPGA')}")
```

Comment:
We should name it something like `FAB_PROJ_VERSION`, to identify, that is really about the project.